src/main/kotlin/app/extractors/CppExtractor.kt

```diff
@@ -32,12 +32,19 @@ class CppExtractor : ExtractorInterface {
         fileContent.forEach {
             val res = extractImportRegex.find(it)
             if (res != null) {
-                val lineLib = res.groupValues.last()
+                val lineLib = res.groupValues
+                                 .last { !it.startsWith(".") && it != ""}
                 imports.add(lineLib)
             }
         }
-
         val libraries = imports.map { MULTI_IMPORT_TO_LIB.getOrDefault(it, it) }
+                               .map { import -> when {
+                                   import.startsWith("Q") -> "Qt"
```

Comment:
Every import that starts with "Q" will be Qt? Is it reasonable?

Reply:
At the moment yes

Reply:
Offline discussion